docs/source/user_guide/benchmarks/surfaces.rst

```diff
@@ -101,3 +101,47 @@ Reference data:
 
 * Same as input data
 * PBE-D3(BJ), MPRelaxSet settings
+
+Elemental Slab Oxygen Adsorption
+===
```

Comment:
```suggestion
================================
```
This needs to be the same length as the title, otherwise it doesn't recognise the heading, unfortunately